docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -70,115 +72,167 @@ ifdef::qs[]
+Welcome to the quick start for {product-long-connectors}. 
+
+In this quick start, you learn how to create a source connector and sink connector and send data to and from {product-kafka}. 
+
+A *source* connector allows you to send data from an external system to {product-kafka}. A *sink* connector allows you to send data from {product-kafka} to an external system. 
+
+endif::[]
+
+ifndef::qs[]
+== Overview
+
+{product-long-kafka} is a cloud service that simplifies the process of running Apache Kafka. Apache Kafka is an open-source, distributed, publish-subscribe messaging system for creating fault-tolerant, real-time data feeds.  
+
+You can use {product-connectors} to configure communication between {product-kafka} instances and external services and applications. {product-connectors} allow you to configure how data moves from one endpoint to another without writing code. 
+
+The following diagram illustrates how data flows from a source of data through a data source connector to a Kafka topic. And how data flows from a kafka topic to a  data sink connector to a data sink.
+
+image::connectors-diagram.png[Illustration of data flow from data source through Kafka to data sink]
+
 
 endif::[]
 
 [id="proc-configuring-kafka-for-connectors_{context}"]
 == Configuring the {product-kafka} instance for use with {product-long-connectors}
 
 [role="_abstract"]
-As a prerequisite for using {product-long-connectors} you need a running {product-kafka} instance. P{lease refer to the Getting started with OpenShift Streams for Apache Kafka for instructions to create a Kafka instance.
+Configure your {product-kafka} for use with {product-long-connectors} by creating  Kafka topics, creating service accounts, and setting up access rules for the service accounts. The number of Kafka topics, service accounts that you create, and the access rules for the service accounts depend on your application. 
+
+Kakfa topics store messages sent by producers (data sources) and make them available to consumers (data sinks). Service accounts allow you to connect and authenticate your Connectors with Kafka instances. Access rules for service accounts define how your Connectors accounts can access and use the associated Kafka instance and topics.
 
-In this step you configure your {product-kafka} for use with {product-long-connectors}. This involves creating topics and setting up access rules for service accounts.
+For this quick start, you create one Kafka topic, named *test*, one service account, and you provide all permissions on the service account.
+
+ifndef::qs[]
+.Prerequisites
+* You're logged in to the {service-url-connectors}[^] web console.
+* You've created a  {product-kafka} instance and the instance is in the *Ready* state.
+For instructions on how to create a Kafka instance, see _Getting started with OpenShift Streams for Apache Kafka_. 
```

Comment:
I think you can actually link to the Getting started quick start: "...see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].